code/game/objects/structures/lattice.dm

```diff
@@ -7,7 +7,7 @@
 	density = FALSE
 	anchored = TRUE
 	armor = list(MELEE = 50, BULLET = 0, LASER = 0, ENERGY = 0, BOMB = 0, BIO = 0, RAD = 0, FIRE = 80, ACID = 50)
-	max_integrity = 50
+	max_integrity = 100000
 	layer = LATTICE_LAYER //under pipes
 	plane = FLOOR_PLANE
 	obj_flags = CAN_BE_HIT | BLOCK_Z_OUT_DOWN
```

Comment:
this is a bad approach, instead of increasing the health please remove the `CAN_BE_HIT` in `obj_flags` whilst adding `INDESTRUCTIBLE` and `NODECONSTRUCT_1`

As just boosting the  health would leave the silent issue of players being able to waste their attacks with possibly REALLY long cooldowns on the lattices
```suggestion
	obj_flags = BLOCK_Z_OUT_DOWN | NODECONSTRUCT_1 | INDESTRUCTIBLE
```

Reply:
Alright, I'll go make that change now.